cursorless-talon/src/targets/primitive_target.py

```diff
@@ -11,7 +11,13 @@
     )
 )
 def cursorless_primitive_target(m) -> PrimitiveTarget:
-    return PrimitiveTarget(
-        getattr(m, "cursorless_mark", None),
-        getattr(m, "cursorless_modifier_list", None),
-    )
+    mark = getattr(m, "cursorless_mark", None)
+    modifiers = getattr(m, "cursorless_modifier_list", None)
+
+    if mark is not None and mark["type"] == "literal":
```

Comment:
This is the closest place where we have both the mark and the modifiers. I try to do as much as possible constructing the modifier in the literal mark. Basically only moving it from a mark to a modifier is done at this stage.